<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlgorithmWidget</class>
 <widget class="QWidget" name="AlgorithmWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>653</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="beginButton">
         <property name="text">
          <string>begin</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="prewButton">
         <property name="text">
          <string>prev</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="stopButton">
         <property name="text">
          <string>stop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="playButton">
         <property name="text">
          <string>play</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="nextButton">
         <property name="text">
          <string>next</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="endButton">
         <property name="text">
          <string>end</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="delay_label">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Delay:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="suffix">
          <string notr="true">  ms</string>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
         <property name="singleStep">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QToolButton" name="checkButton">
         <property name="text">
          <string>Check solution</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="showButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Show solution</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="checkSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QToolButton" name="prew_stepButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>prew step</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="next_stepButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>next step</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="stepSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="Formal">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="formalTab">
        <attribute name="title">
         <string>formal</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="AlgorithmView" name="treeView">
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>400</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="informalTab">
        <attribute name="title">
         <string>informal</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QTextEdit" name="textEdit_2"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AlgorithmView</class>
   <extends>QTreeView</extends>
   <header>algorithms/algorithmview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
